node/src/cli.rs

```diff
@@ -149,13 +137,11 @@ pub enum Subcommand {
 
     /// Export the genesis state of the parachain.
     #[cfg(feature = "parachain")]
-    #[clap(name = "export-genesis-state")]
-    ExportGenesisState(cli_parachain::ExportGenesisStateCommand),
+    ExportGenesisState(cumulus_client_cli::ExportGenesisStateCommand),
 
     /// Export the genesis wasm of the parachain.
     #[cfg(feature = "parachain")]
-    #[clap(name = "export-genesis-wasm")]
-    ExportGenesisWasm(cli_parachain::ExportGenesisWasmCommand),
+    ExportGenesisWasm(cumulus_client_cli::ExportGenesisWasmCommand),
```

Comment:
Moonbeam [uses still the macro `#[clap(name = "export-genesis-state")]`](https://github.com/PureStake/moonbeam/blob/c44fd0c46ef260543745998ff18d7e666d8c139a/node/cli/src/cli.rs#L31-L37) with the polkadot version 0.9.26.

Reply:
Am I missing something here?

Reply:
perhaps that is correct as cumulus may not be updated from moonbeam.

Reply:
I guess it's okay if Acala doesn't use it [here](https://github.com/AcalaNetwork/Acala/blob/4bd73a31da9cc83dab8e13d10d9c4b34770d8476/node/cli/src/cli.rs#L30-L34) with 0.9.26.